upload/tests/views/test_bundle_analysis.py

```diff
@@ -246,3 +253,98 @@ def test_upload_bundle_analysis_github_oidc_auth(
         format="json",
     )
     assert res.status_code == 201
+
+
+@pytest.mark.django_db(databases={"default", "timeseries"})
+def test_upload_bundle_analysis_measurement_datasets_created(
```

Comment:
jw, is it possible to parametrize this and the other test with the pytest.mark.parametrize() decorator?


https://github.com/codecov/codecov-api/blob/7fc9bc556c81a50855f0ab8b2402fd0d85be8cc3/services/tests/test_repo_providers.py#L41